agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4555,9 +4555,13 @@ bool slave_thread::handle_channel_selection_request(Socket *sd, ieee1905_1::Cmdu
         return false;
     }
 
-    if (!switch_required && !power_limit_received) {
+    if (!switch_required) {
+        LOG(INFO) << "Channel switch not required, sending operating channel report";
         send_operating_channel_report();
```

Comment:
Is this temporary solution? If power limit was received, channel report should contain new power limit.

Reply:
Good point, will fix it.

Reply:
Yes this is a temporary solution so we can pass certification. I added a comment and a TODO message linking this to https://github.com/prplfoundation/prplMesh/issues/725